Dockerfile

```diff
@@ -97,11 +97,11 @@ USER root
 
 ### JAVASCRIPT
 
-# Install Node 14.0 and npm v7
+# Install Node 14.0 and npm v8
 RUN curl -sL https://deb.nodesource.com/setup_14.x | bash - \
```

Comment:
```suggestion
RUN curl -sL https://deb.nodesource.com/setup_16.x | bash - \
```

Reply:
👍  on this change but might be worth splitting this up into a separate PR to make it easier to isolate potential regressions.